uiowa/uiowa
# I3060 accordion updates

Initial PR for tracking progress on #3060 


# How to test:
1. Sync down the sandbox site, you should see some imported config for a few new fields
2. `uli` in to the site and create a page
3. Create a collection bundle, and fill out some collection items. I've found 3 to be a good number.
    a. Verify that there is an option on collection item called `Automatically open`.
4. Verify that the `View mode` radio field is just above the collection items
5. Verify that if you select `Accordion` in the `View mode` that another field, `Accordion Type` appears just under it.
6. Verify that you can select either `Multi select` or `Single select` in the `Accordion type` field, and that the help text for this field describes how they work, and provides extra instructions for properly using the `Single select` option.
7. Verify that if you select `Multi select`: 
    a. Setting the `Automatically open` on any amount of collection items makes them automatically open when loaded. 
    b. Verify that many accordion items can be open at the same time.
8. Verify that if you select `Single select`: 
    a.  Verify that only one accordion can be open at a time after the first opening of an accordion.
    b. For the completion of this issue, it was deemed that with `Single select` it was up to the users to only have one item set to `Automatically open`. If they set multiple collection items to be `Automatically open` with `Single select`, the set items will automatically open, and then once a closed item is opened, the `Single select` collapsing functionality will activate. See example below.
    
![aded300063e8bad71c341e17f06fa28d](https://user-images.githubusercontent.com/23122142/110169954-12a59880-7dbf-11eb-8318-163ee2281504.gif)
